tock/employees/tests/test_views.py

```diff
@@ -146,7 +146,7 @@ def test_UserFormViewPostForAdmin(self):
         self.assertEqual(response.status_code, 200)
         # Check if data was changed
         # Check that data was changed
-        user_data = UserData.objects.first()
+        user_data = UserData.objects.get(user__first_name='Regular')
```

Comment:
This might cause an issue if more than 1 object exists with the same first name.  I know this is the test database, but we should still be careful with these assumptions and make sure we explicitly return or find a single record.